tectonic/xetex-xetex0.c

```diff
@@ -8283,14 +8186,11 @@ xetex_scan_dimen(bool mu, bool inf, bool shortcut, bool requires_units)
 
                 while (scan_keyword("l")) {
                     if (cur_order == FILLL) {
-                        if (file_line_error_style_p)
-                            print_file_line();
-                        else
-                            print_nl_cstr("! ");
-                        print_cstr("Illegal unit of measure (");
-                        print_cstr("replaced by filll)");
+                        error_here_with_diagnostic("Illegal unit of measure (replaced with filll)");
+                        capture_to_diagnostic(NULL);
+
                         help_ptr = 1;
-                        help_line[0] = "I dddon't go any higher than filll.";
+                        help_line[0] = "I don't go any higher than filll.";
```

Comment:
I think this is intentional -- looks like wordplay from Knuth inspired by "filll". For compatibility, I'd like to keep this as-is.

Reply:
Sounds good, I've pushed a change for this.